src/Symfony/Bundle/FrameworkBundle/Resources/config/web.php

```diff
@@ -60,6 +63,14 @@
             ])
             ->tag('controller.argument_value_resolver', ['priority' => 100])
 
+        ->set('argument_resolver.request_payload', RequestPayloadValueResolver::class)
+            ->args([
+                service('serializer')->nullOnInvalid(),
+                service('serializer')->nullOnInvalid(),
+                service('validator')->nullOnInvalid(),
+            ])
+            ->tag('controller.argument_value_resolver', ['priority' => 100])
```

Comment:
I'd like waiting for #48992 and registering this using `controller.pinned_value_resolver`

Reply:
I'm afraid that we won't merge it before feature freeze for 6.3

Reply:
@nicolas-grekas done :heavy_check_mark: